packages/extension/src/background/price.ts

```diff
@@ -0,0 +1,95 @@
+import { number } from "starknet"
+
+import { defaultNetwork } from "../shared/networks"
+import { Token, UniqueToken, equalToken, getFeeToken } from "../shared/token"
+import { StaleWhileRevalidateCache } from "./swr/types"
+import { fetchWithTimeout } from "./utils/fetchWithTimeout"
+
+type BaseCorrency = "usd"
```

Comment:
technically it's actually called the quote currency but whatever
```suggestion
type BaseCurrency = "usd"
```